dotorg/dotorg-support-dashboard.user.js

```diff
@@ -16,6 +16,7 @@
 // @require      https://unpkg.com/dayjs@1.8.21/plugin/customParseFormat.js
 // @downloadURL  https://github.com/the-events-calendar/tampermonkey-scripts/raw/main/dotorg/dotorg-support-dashboard.js
 // @updateURL    https://github.com/the-events-calendar/tampermonkey-scripts/raw/main/dotorg/dotorg-support-dashboard.js
```

Comment:
The URLs point to the old filename.

Reply:
Question:
Above on the `@match` and `@exclude` lines, doesn't the `https://*.wordpress.org/support/*` version also include the domain without subdomain, namely `https://wordpress.org/support/*`?